.github/workflows/linting-and-unittests.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
Since we support 3.9, we can change it to "We support Python 3 (>=3.7)." in the `Installation` section in [README.md](https://github.com/Unity-Technologies/datasetinsights/blob/master/README.md#installation).